modules/kafka/kafka-dest-worker.c

```diff
@@ -340,6 +351,26 @@ _thread_init(LogThreadedDestWorker *s)
   return log_threaded_dest_worker_init_method(s);
 }
 
+static gboolean
+kafka_dest_worker_connect(LogThreadedDestWorker *s)
+{
+  KafkaDestWorker *self = (KafkaDestWorker *)s;
+  KafkaDestDriver *owner = (KafkaDestDriver *) self->super.owner;
```

Comment:
I do not see an expression that I could use in that assert. (Asserting on either running in `worker#0` or `sync-send(yes)` would not do justice.) Using this in case of multiple worker would require additional consideration.

Reply:
Okay, thanks for checking.